src/docs/DocsWrapper.tsx

```diff
@@ -22,14 +27,30 @@ const paddingIncrement = 2
 
 export default function DocsWrapper({ children }: DocsWrapperProps): ReactElement {
   const theme = useTheme()
-  const { pathname } = useRouter()
+  const router = useRouter()
```

Comment:
Is there a reason for reverting from destructuring? I tend to prefer destructuring as it reduces the amount of dot notation being used throughout the code.

Reply:
We need to use router itself to change page, L#114 for example.